package.json

```diff
@@ -36,14 +36,14 @@
     "@types/mocha": "^7.0.0",
     "@types/node": "^10.5.2",
     "@types/sinon": "^7.0.5",
+    "c8": "^7.0.0",
     "codecov": "^3.0.4",
-    "gts": "^1.0.0",
+    "gts": "2.0.0-alpha.5",
```

Comment:
Could just pin this to `next` for now to always float to the latest

Reply:
I think we're getting very close to being able to just cut `2.x.x`, I'd rather we're explicit about version, and let renovate do its thing soon.